src/overlay.md

```diff
@@ -265,6 +268,53 @@ actions:
 
 This approach allows inversion of control as to where the Overlay updates apply to the target document itself.
 
+#### Copy example
+
+Copy actions behave similarly to update actions but source the node to from the document being transformed. Copy actions MAY be combined with update or remove actions to perform more advanced transformations like moving or renaming nodes.
```

Comment:
Perhaps "used in sequence with" rather than "combined with" since the fields are mutually exclusive and can't be combined - so this could be confusing.

Reply:
updated